lib/network/validation/proto/validation.proto

```diff
@@ -12,11 +12,26 @@ service Validation {
     // CheckBandwidth executes a bandwidth network test
     rpc CheckBandwidth(CheckBandwidthRequest) returns (CheckBandwidthResponse);
 
+    // CheckDisks executes performance test for the specified disks
+    rpc CheckDisks(CheckDisksRequest) returns (CheckDisksResponse);
+
     // Validate validatest this node against the requirements
     // from a manifest.
     rpc Validate(ValidateRequest) returns (ValidateResponse);
 }
 
+// CheckDisksRequest describes a disks performance test request.
+message CheckDisksRequest {
+    // Spec is a fio job spec describing the test to perform.
+    bytes spec = 1;
+}
+
+// CheckDisksResponse is the result of the disk performance test.
+message CheckDisksResponse {
+    // Result is the fio job result output in json format.
+    bytes result = 1;
```

Comment:
Why not define it in protobuf format?

Reply:
Yeah, I was actually considering that but just wasn't sure how the output changes based on the fio configuration and whether it's possible to describe it with a single struct. But I think it would be better, yes, so I'll do that for our use-case.